goawk.go

```diff
@@ -185,11 +210,19 @@ argsLoop:
 				cpuprofile = arg[12:]
 			case strings.HasPrefix(arg, "-memprofile="):
 				memprofile = arg[12:]
+			case strings.HasPrefix(arg, "-covermode="):
+				covermode = arg[11:]
```

Comment:
I know I did this already, but let's change these long ones to use `arg[len("-covermode="):]` instead of `arg[11:]` and suchlike. (But leave the `arg[2:]` ones -- they're obvious.)